src/patterns/dosdp-patterns/OMIM_phenotypic_series

```diff
@@ -0,0 +1,36 @@
+pattern_name: OMIM_phenotypic_series
+
+description: >-
+
+This patterrn is meant to be used for OMIM phenotypic series (OMIMPS), which are represented as grouping classes in Mondo. Note: 
+- every instance of this metaclass should be equivalent to (via annotated xref) to something in OMIMPS namespace
```

Comment:
@matentzn not urgent but to think about for future pattern calls. How do we encode this? Do we extend the dosdp formalism? Until it ends up incorporating half of shex? this is why I think we need to ultimately subsume within a more general framework